src/mono/mono/eventpipe/ep-rt-mono.h

```diff
@@ -980,6 +980,19 @@ ep_rt_config_value_get_circular_mb (void)
 	return circular_mb;
 }
 
+static
+inline
+bool
+ep_rt_config_value_get_use_portable_thread_pool (void)
```

Comment:
Mono supports portable thread pool only, this should just return true

Reply:
Fixed.